deploy/kubernetes/resources/gameon-impl-statefulset.json

```diff
@@ -0,0 +1,95 @@
+{
+  "apiVersion": "apps/v1beta1",
+  "kind": "StatefulSet",
```

Comment:
Strictly speaking, we don't need to use a `StatefulSet` for these services, since they do not use an Akka cluster, but maybe it's simpler to stick with the known solution.